app/components/picker-section-li.js

```diff
@@ -0,0 +1,22 @@
+import Ember from 'ember';
+
+export default Ember.Component.extend({
+  tagName: 'li',
+
+  init() {
+    this._super(...arguments);
+
+    Ember.run.next(() => {
+      this.set('loaded', true);
+    });
+  },
+
+  properties: Ember.computed('service.id', 'service.name', 'service.isSelected', function() {
+    return {
+      selected: !this.get('service.isSelected'),
```

Comment:
Negating the `this.get('service.isSelected')` value here to set the `selected` property is a hack to overcome a race condition in which the `properties` computed value here doesn't get updated before we call `trackEvent` on the segment service with these properties elsewhere. The intended effect is to report the service's new "selected" status even if these properties haven't been updated yet, but instead of the hack, the race condition itself should be resolved, perhaps by using `Ember.run.later` to delay `trackEvent` internal logic.